docs/Deployment/ManualDeploymentExtended.md

```diff
@@ -821,13 +821,16 @@ If it doesn't start correctly stop the service and run the application directly
 ~$ cd ~; pushd ./src/NBXplorer; ./run.sh; popd
 ```
 
-#####  Update
+##### Update
 
 Updating could break things. Be careful on a live system.
 
 ```bash
+# Stop the service
 ~$ sudo systemctl stop nbxplorer
-~$ cd ~; pushd ~/src/NBXplorer; git pull; ./build.sh; popd;
+# Checkout and build latest tag
+~$ cd ~; pushd ~/src/NBXplorer; it checkout $(git tag --sort -version:refname | awk 'match($0, /^v[0-9]+\./)' | head -n 1); ./build.sh; popd;
```

Comment:
did not try the commands but there seems to be a `g` missing in the git command

Reply:
Thanks!

Reply:
```suggestion
~$ cd ~; pushd ~/src/NBXplorer; git checkout $(git tag --sort -version:refname | awk 'match($0, /^v[0-9]+\./)' | head -n 1); ./build.sh; popd;
```